go/vt/wrangler/keyspace.go

```diff
@@ -145,9 +146,7 @@ func (wr *Wrangler) printShards(ctx context.Context, si []*topo.ShardInfo) error
 				wr.Logger().Printf("        %v\n", row)
 			}
 		}
-		if len(si.ServedTypes) != 0 {
-			wr.Logger().Printf("      Served Types: %v\n", si.ServedTypes)
-		}
+		wr.Logger().Printf("      Served Types: %v\n", si.IsMasterServing)
```

Comment:
The message "Served Types" isn't accurate for this log message anymore. Should probably be something like "Is Master Serving"

Reply:
Fixed in: https://github.com/vitessio/vitess/pull/4760